src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle'
+  color: Color
+  getArea: ()=>{}
```

Comment:
The function 'getArea' in the interface 'Figure' needs to specify a return type. It should return a number.